specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/v1/OperationalInsights.json

```diff
@@ -88,7 +88,7 @@
         }
       },
       "post": {
-        "operationId": "Query",
+        "operationId": "Query_Execute",
```

Comment:
The changes in stable API are potential SDK breaking changes

Reply:
I had been doing work to clean up the operation groups as part of a general effort towards stable/ v1.0.0 versions of the data plane SDKs (we have preview versions for C#, Node, Java, and Python). This isn't strictly necessary and I could revert it, but is in line with what I've done in the other api-versions/with App Insights as well.